/**:
  ros__parameters:
    # General parameters
    debug: False
    use_path_planner: true
    # Path planner parameters
    path_planner_timeout: 20.0
    navigation_map_path: "mapas/Alamillo95x216"
    binary_dilation: 2
    # WQP Sensors parameters
    wqp_sensor_USB_string: "/dev/SENSOR"
    wqp_sensor_baudrate: 115200
    wqp_sensor_measurement_frequency: 1.0
    wqp_sensor_timeout: 6.0
    # Sonar parameters
    sonar_USB_string: "/dev/SONAR"
    sonar_measurement_frequency: 1.0
    sonar_baudrate: 115200
    # Server parameters
    internet_loss_timeout: 30.0
    mqtt_addr: "127.0.0.1"
    mqtt_port: 1883
    mqtt_user: "asv3" # TODO: Change to avoid collisions
    mqtt_password: "azkenmuga"
    # Database parameters
    database: wqp
    db_user: root
    db_password: password
    db_port: '3306'
    db_host: "127.0.0.1"



